src/main/java/pl/javastart/task/TaxCalculate.java

```diff
@@ -0,0 +1,12 @@
+package pl.javastart.task;
+
+public class TaxCalculate {
+    double taxPrice(Product product) {
+        double price = switch (product.getCategory()) {
+            case "Usługi" -> product.getPriceNet() * 1.23;
+            case "Produkt spożywczy" -> product.getPriceNet() * 1.05;
+            default -> product.getPriceNet() * 1.08;
```

Comment:
Z 1.23, 1.05 i 1.00 zrób dobrze nazwane stałe i będzie ok. Najlepiej jakby przy deklarowaiu wysokości stawki VAT użyć wartości 23, albo 0.23, jest to bardziej intuicyjne.